### DISCUSSION TITLE
Will existing gallery get uploaded?

### DISCUSSION THREAD

**Discussion by Pheggas** [immich-app/immich#2647] [Q&A] [answered]
Hello. I have existing gallery from Syncthing and i wonder if the gallery will get uploaded once again (so the same file will exist in two locations) or it will just make thumbnails and sidecars and the originals will not get duplicated?

**Comment by alextran1502** [comment]
> will get uploaded once again

Where did you perform the initial upload from?

**Reply by Pheggas** [reply]
It is just a theoretical question. I didn't deploy it yet but i'm about to do so.

**Reply by alextran1502** [reply, marked as answer]
Ah so you will be uploading from Syncthing location? If so, the files will exist in both location since Immich will need to ingest the file

**Reply by Pheggas** [reply]
Alright thank you very much! I know it is not usable for production yet but if i will do regular backups of Immich itself, is it suitable as the only "backuping way"?

Because it is a waste of space to have 2 copies of same data on NAS

**Reply by alextran1502** [reply]
I use Immich myself as a production backup solution for my wife and me. However, as far as backup goes, we also have an offsite backup. And this should be applied to any scenario or any software you use to view the photos and videos.

To answer your question, after moving your photos and videos to Immich, you can remove those in Syncthing because it is basically the same files stored on your file system.

**Reply by Pheggas** [reply]
Yup. I'm basically going to move all photos from syncting (and as i'm using syncthing only for photo backup, i can remove the whole container - app). I have solid offsite backup plan so i think i should be safe. Thank you once more and good luck with Immich!
